1_space_odyssey/index.html

```diff
@@ -10,7 +10,7 @@
 </head>
 <body class="fade-in">
     
-    <script src="./js/mannuel-9000/app.js"></script>
+    <script src="../js/christoffer-9000/app.js"></script>
```

Comment:
please leave my import script to my mannuel-9000 app :-)

also only need one dot please, not two.
like so:
`src="./js/christoffer-9000/app.js"`